PathPlanning/RRT/rrt_with_sobol_sampler.py

```diff
@@ -30,6 +30,7 @@
 import random
 import matplotlib.pyplot as plt
 import numpy as np
+import sys
```

Comment:
Thank you for your PR. Is it possible to add a test to run this script?

Reply:
Ignore this comment and please merge latest master branch. Another PR fixed this.

Reply:
Thanks - I rebased these changes on top of the latest master branch.